typespec-tests/src/main/java/com/_specs_/azure/core/basic/implementation/TwoModelsAsPageItemsImpl.java

```diff
@@ -369,6 +369,9 @@ public PagedIterable<BinaryData> listSecondItem(RequestOptions requestOptions) {
     }
 
     /**
+     * Two operations with two different page item types should be successfully generated. Should generate model for
```

Comment:
This comes from operation's doc.
https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/azure/core/basic/main.tsp#L317